enterprise/setup/index.md

```diff
@@ -0,0 +1,323 @@
+Jul  2 07:15:16 aa84f55aceba syslog-ng[160]: syslog-ng starting up; version='3.13.2'
+Jul  2 07:15:17 aa84f55aceba cron[180]: (CRON) INFO (pidfile fd = 3)
+Jul  2 07:15:17 aa84f55aceba cron[180]: (CRON) INFO (Running @reboot jobs)
+ set_config 
+------------
+ 
+(1 row)
+
+Jul  2 07:15:22 aa84f55aceba syslog-ng[160]: syslog-ng shutting down; version='3.13.2'
+```
+
+Next, create the initial user:
+
+```
+docker run --env-file pganalyze.env quay.io/pganalyze/enterprise:[version] rake db:seed
+```
+
+Make sure to note down the default user email and password:
+
+```
+Jul  2 07:15:29 8bc5a4c0e6b6 syslog-ng[150]: syslog-ng starting up; version='3.13.2'
+Jul  2 07:15:30 8bc5a4c0e6b6 cron[167]: (CRON) INFO (pidfile fd = 3)
+Jul  2 07:15:30 8bc5a4c0e6b6 cron[167]: (CRON) INFO (Running @reboot jobs)
+I, [2020-07-02T07:15:35.006455 #159]  INFO -- : *****************************
+I, [2020-07-02T07:15:35.006520 #159]  INFO -- : *** INITIAL ADMIN CREATED ***
+I, [2020-07-02T07:15:35.006538 #159]  INFO -- : *****************************
+I, [2020-07-02T07:15:35.006551 #159]  INFO -- : 
+I, [2020-07-02T07:15:35.006563 #159]  INFO -- : *****************************
+I, [2020-07-02T07:15:35.006599 #159]  INFO -- : Email:    admin@example.com
+I, [2020-07-02T07:15:35.006619 #159]  INFO -- : Password: abcdefabcdef
+I, [2020-07-02T07:15:35.006636 #159]  INFO -- : *****************************
+I, [2020-07-02T07:15:35.006648 #159]  INFO -- : 
+I, [2020-07-02T07:15:35.006660 #159]  INFO -- : Use these credentials to login and then change email address and password.
+Jul  2 07:15:35 8bc5a4c0e6b6 syslog-ng[150]: syslog-ng shutting down; version='3.13.2'
+```
+
+### Step 4: Start Docker Image & Login to pganalyze
+
+You can now start the Docker image in the background using this command:
+
+```
+docker run -d --name pganalyze --env-file pganalyze.env -p 8080:5000 quay.io/pganalyze/enterprise:[version]
+```
+
+This will make the web interface accessible on the server's address, on port 8080. You should see a site like this when you open your web browser for that address and port:
+
+![](login.png)
+
+Please now login to the pganalyze interface using the generated credentials you've seen earlier when setting up the database. If successful you should see this screen:
+
+![](create_organization.png)
+
+If authentication does not work, or you see an error message, please check the container's logs using `docker logs`.
+
+Choose an organization name of your choice (typically your company name), and then click "Add Server" on the next screen. You should now see this form:
+
+![](add_server.png)
+
+### Step 5: Preparing Your PostgreSQL Database for Monitoring
+
+Before you can add a database to the pganalyze installation, you'll need to enable the `pg_stat_statements` extension on it.
+
+This extension is bundled with PostgreSQL and allows you detailed query monitoring. Its overhead on the database system is minimal (although not zero), and it is required to use pganalyze.
```

Comment:
> (although not zero)

This doesn't really add anything, since "minimal" is already saying the same thing--we should either drop it or discuss the edge case of when this actually does matter (probably the former).